### PR TITLE
WT-12047 improve block_ext logs

### PR DESCRIPTION
**Pull request by y123456yz**
some log perfect, this can help us to understand the code.

### CODE REVIEW

**Comment by y123456yz** [comment]
https://jira.mongodb.org/browse/WT-12047

**Review by ruby-oujo — APPROVED** [review]
LGTM

File: src/btree/bt_handle.c [hunk 1]
```diff
@@ -323,7 +323,8 @@ __btree_conf(WT_SESSION_IMPL *session, WT_CKPT *ckpt, bool is_ckpt)
         maj_version = cval.val;
         WT_RET(__wt_config_gets(session, cfg, "version.minor", &cval));
         min_version = cval.val;
-        __wt_verbose(session, WT_VERB_VERSION, "%" PRId64 ".%" PRId64, maj_version, min_version);
+        __wt_verbose(
+          session, WT_VERB_VERSION, "btree version:%" PRId64 ".%" PRId64, maj_version, min_version);
```
Comment (ruby-oujo) [inline src/btree/bt_handle.c:327]: NP: Can we add a space here: "btree version: %", before "%" please.

Comment (y123456yz) [reply src/btree/bt_handle.c]: done, thanks, please review again.

**Comment by ruby-oujo** [comment]
evergreen merge

**Pull request merged**